docs/libcurl/opts/CURLINFO_RESPONSE_CODE.3

```diff
@@ -28,15 +28,16 @@ CURLINFO_RESPONSE_CODE \- get the last response code
 
 CURLcode curl_easy_getinfo(CURL *handle, CURLINFO_RESPONSE_CODE, long *codep);
 .SH DESCRIPTION
-Pass a pointer to a long to receive the last received HTTP, FTP or SMTP
-response code. This option was previously known as CURLINFO_HTTP_CODE in
-libcurl 7.10.7 and earlier. The stored value will be zero if no server
-response code has been received. Note that a proxy's CONNECT response should
+Pass a pointer to a long to receive the last received HTTP, FTP, SMTP or
+LDAP (openldap only) response code. This option was previously known as
+CURLINFO_HTTP_CODE in libcurl 7.10.7 and earlier.
+The stored value will be zero if no server response code has been received.
+Note that a proxy's CONNECT response should
 be read with \fICURLINFO_HTTP_CONNECTCODE(3)\fP and not this.
 
 Support for SMTP responses added in 7.25.0.
 .SH PROTOCOLS
-HTTP, FTP and SMTP
+HTTP, FTP, SMTP and LDAP
```

Comment:
Maybe add  a mention in which version the LDAP support is added?

Reply:
As `x.xx.x` for now, then! I'm still waiting for the binary SASL PR to be pulled in order to complete this PR, as the openldap SASL code is ready here.
But maybe you're interested in having this without SASL?

Reply:
This is a PR, I reviewed it and gave you my comments...  If your binary SASL work is ready for review too, feel free to request a review here on github to signal that. I often get lost a little in an ocean of github issues/PRs.

Reply:
Done, with x.xx.x as version number. To be adjusted before pull.